java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -522,6 +524,94 @@ public void getdel() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void getex() {
+        // setup
+        String key = "testKey";
+        String value = "testValue";
+        CompletableFuture<String> testResponse = new CompletableFuture<>();
+        testResponse.complete(value);
+        when(commandManager.<String>submitNewCommand(eq(GetEx), eq(new String[] {key}), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.getex(key);
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(value, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void getex_options() {
+        // setup
+        GetExOptions options1 = GetExOptions.Seconds(10L);
+        GetExOptions options2 = GetExOptions.Milliseconds(1000L);
+        GetExOptions options3 = GetExOptions.UnixSeconds(10L);
+        GetExOptions options4 = GetExOptions.UnixMilliseconds(1000L);
+        GetExOptions options5 = GetExOptions.Persist();
+
+        String[] arguments1 = new String[] {"key", "EX", "10"};
+        String[] arguments2 = new String[] {"key", "PX", "1000"};
+        String[] arguments3 = new String[] {"key", "EXAT", "10"};
+        String[] arguments4 = new String[] {"key", "PXAT", "1000"};
+        String[] arguments5 = new String[] {"key", "PERSIST"};
```

Comment:
I'd make it as a separate UT for options only - without mocks, see `xadd_with_options_to_arguments` for example.
So you can have a UT with a mock to test the command and the second parametrized UT which confirm that options converted to command args properly.